### DISCUSSION TITLE
我啥也看不到

### DISCUSSION THREAD

**Discussion by hanshui2077** [share121/inter-knot#797] [其他]
只能在girhub页面上看到帖子和评论

**Comment by Lanxey** [comment]
想看啥

**Reply by hanshui2077** [reply]
网页上只有一个加载中

**Comment by SDCOM-0415** [comment]
**你想看什么啊，孩子**

**Reply by hanshui2077** [reply]
![IMG_6138](https://github.com/user-attachments/assets/e0f95a08-f056-4395-a924-064d18864b12)
网站上只能看到这样，我在直接从github进聊天

**Comment by aaa00073** [comment]
难道还有除了帖子和评论以外的东西吗？
